developer_guides/cmake.rst

```diff
@@ -54,9 +61,21 @@ MEU_PRIVATE_KEY
       # Example
       cmake [...] -DMEU_PRIVATE_KEY=/path/to/meu/private-key.pem [...]
 
+MEU_OPENSSL

```

Comment:
Can we mark this as optional and as Intel and Windows only since we now are merging PRs for non Intel architectures.

Reply:
All variables are optional. Only `TOOLCHAIN` and `ROOT_DIR` are needed if  you build FW